read.go

```diff
@@ -46,7 +46,9 @@ func (i *UI) read(opts *readOptions) (string, error) {
 				resultErr = fmt.Errorf("failed to read the input: %s", err)
 			}
 
-			resultStr = strings.TrimSuffix(line, "\r\n")
+			resultStr = strings.TrimSuffix(line, LINE_SEP)
```

Comment:
@tcnksm I'm not sure if this is idiomatic, but this will allow platform specific line separators. The 'problem' is that most of the test cases are hard coded with `\n`, hence on line#51 below I'm also doing a second trimming of `\n`. This both causes the tests to pass, as well as confirms to work across platforms (at least tested on windows+OSX)